### ISSUE TITLE
Nightly build failed on fontawesome auth

### ISSUE THREAD

**Issue by dweiss** [carrot2/carrot2#153]
```
> Task :dcs:contexts:frontend:yarnInstall
  ➤ │ @fortawesome/fontawesome-pro@npm:5.15.4: Invalid authentication (as an unknown user)
  ➤ │ @fortawesome/fontawesome-common-types@npm:0.2.36: Invalid authentication (as an unknown user)
  ➤ │ @fortawesome/fontawesome-svg-core@npm:1.2.36: Invalid authentication (as an unknown user)
  ➤ │ @fortawesome/free-brands-svg-icons@npm:5.15.4: Invalid authentication (as an unknown user)
  ➤ │ @fortawesome/pro-duotone-svg-icons@npm:5.15.4: Invalid authentication (as an unknown user)
  ➤ │ @fortawesome/free-solid-svg-icons@npm:5.15.4: Invalid authentication (as an unknown user)
  ➤ │ @fortawesome/pro-light-svg-icons@npm:5.15.4: Invalid authentication (as an unknown user)
  ➤ │ @fortawesome/pro-regular-svg-icons@npm:5.15.4: Invalid authentication (as an unknown user)
  ➤ │ @fortawesome/react-fontawesome@npm:0.1.15::__archiveUrl=https%3A%2F%2Fnpm.fontawesome.com%2F%40fortawesome%2Freact-fontawesome%2F-%2F0.1.15%2Freact-fontawesome-0.1.15.tgz: Invalid authentication (as an unknown user)
  ➤ │ @fortawesome/react-fontawesome@npm:0.1.16: Invalid authentication (as an unknown user)
```

**Comment by dweiss** [comment]
https://github.com/carrot2/carrot2/actions/runs/2709504508

I don't know what this can be or whether it's temporary.

**Comment by stanislawosinski** [comment]
This is related to expired FontAwesome subscription. I'll see if I can replace the icons with some open alternative.

**Comment by dweiss** [comment]
Ah, ok. Yeah, let's replace them - it'll be easier to handle too, I guess.

**Comment by stanislawosinski** [comment]
Fixed as part of #154.
